esrally/track/params.py

```diff
@@ -1122,6 +1203,10 @@ def read_bulk(self):
 register_param_source_for_operation(track.OperationType.DeleteDataStream, DeleteDataStreamParamSource)
 register_param_source_for_operation(track.OperationType.CreateIndexTemplate, CreateIndexTemplateParamSource)
 register_param_source_for_operation(track.OperationType.DeleteIndexTemplate, DeleteIndexTemplateParamSource)
+register_param_source_for_operation(track.OperationType.CreateComponentTemplate, CreateComponentTemplateParamSource)
+register_param_source_for_operation(track.OperationType.DeleteComponentTemplate, DeleteComponentTemplateParamSource)
+register_param_source_for_operation(track.OperationType.CreateComposableTemplate, CreateComposableTemplateParamSource)
+register_param_source_for_operation(track.OperationType.DeleteComposableTemplate, DeleteIndexTemplateParamSource)
```

Comment:
Small note - we use existing `DeleteIndexTemplateParamSource` for `DeleteComposableTemplate`- didn't seem worth duplicating.

Reply:
Ok for me.